### DISCUSSION TITLE
Speaker battery

### DISCUSSION THREAD

**Discussion by Kle0git** [suchmememanyskill/CYD-Klipper#16] [Ideas]
Hello
First of all, thanks for the project, it turned out to be a very useful device)
There is a suggestion, since the board has contacts for the speaker, is it possible to use them for notification, for example, what would the printer notify when printing is complete?

I also installed a battery for my screen, which allows me to take it further than the printer. Is it possible to display the charge on the battery? You may need to solder resistors, because... Initially the screen is switched to stable 5V

**Comment by suchmememanyskill** [comment]
I'd prefer to keep this project scoped to just the board, so no additional accessories like a battery or speaker attached. I currently have no interest in implementing this
